docs/release-notes.md

```diff
@@ -30,16 +33,24 @@ import RedisDomainCaution from '@site/docs/self-hosted-appcircle/configure-serve
-- All publish artifacts including the binary, metadata details, screenshots, App Store Connect information, and more can now be downloaded a zip file. <PublishBadge/> <CloudBadge/> <SelfHostedBadge/>
+- All Publish artifacts including the binary, metadata details, screenshots, App Store Connect information, and more can now be downloaded a zip file. <PublishBadge/> <CloudBadge/> <SelfHostedBadge/>
 - When [inviting a new user to an organization](/account/my-organization#managing-the-team-under-an-organization), a warning message will now be displayed if an invalid or already in-use email address is entered. <AccountBadge/> <CloudBadge/> <SelfHostedBadge/>
+- Existing and newly invited organization members can now be searched by e-mail filter within the Team Management area. <AccountBadge/> <CloudBadge/> <SelfHostedBadge/>
+- A warning message will now be displayed when attempting to delete a provisioning profile that is already in use for a build profile. This allows the user to see the affected profiles and navigate to their configuration directly. Alternatively, the provisioning profile can be force deleted by the user. <SigningIdentitiesBadge/> <CloudBadge/> <SelfHostedBadge/>
+- When a Provisioning Profile within the Apple Profiles has a mismatched Apple Certificate, a tool tip type warning message will be displayed for the user. <SigningIdentitiesBadge/> <CloudBadge/> <SelfHostedBadge/>
+- Apple Provisioning Profile file upload now supports uploading multiple files at once. <SigningIdentitiesBadge/> <CloudBadge/> <SelfHostedBadge/>
+- Multiple Provisioning Profiles can now be force deleted by the users. <SigningIdentitiesBadge/> <CloudBadge/> <SelfHostedBadge/>
 
 ### 🐞 Fixes
 
 - A behaviour difference between Appcircle UI and App Store Connect while registering Bundle IDs has been fixed. <PublishBadge/> <CloudBadge/> <SelfHostedBadge/>
 - An issue causing indicator truncation while retrieving provisioning profiles in the Publish Module has been fixed. <PublishBadge/> <CloudBadge/> <SelfHostedBadge/>
 - An issue has been fixed where, if no images are present in the metadata screenshot section for different localization settings, images from the primary language were not being displayed for guidance and explanatory purposes. <PublishBadge/> <CloudBadge/> <SelfHostedBadge/>
+- The Publish profile header will no longer display the latest store status if the binary is unmarked as a release candidate. <PublishBadge/> <CloudBadge/> <SelfHostedBadge/>
 - The errors that occurred on some types of projects during [Android versioning](https://docs.appcircle.io/versioning#understanding-android-versioning) were fixed, and several improvements were made to check for invalid versioning. <VersioningBadge/> <CloudBadge/> <SelfHostedBadge/>
+- An issue has been fixed where build profile cards displayed some build results as text instead of an icon. <BuildBadge/> <CloudBadge/> <SelfHostedBadge/>
```

Comment:
```suggestion
- Fixed an issue where build profile cards displayed some build results as text instead of an icon. <BuildBadge/> <CloudBadge/> <SelfHostedBadge/>
```